src/plot/axis/axis-title.js

```diff
@@ -23,6 +23,32 @@ import React from 'react';
 import {ORIENTATION} from 'utils/axis-utils';
 
 const {LEFT, RIGHT, TOP, BOTTOM} = ORIENTATION;
+const transformation = (width, height) => ({
+  [LEFT]: {
+    x: 16,
```

Comment:
Essentially, yes - I just converted the em values given to an approximation in pixels (assuming 1em === 16px)

Just realised that I messed up the top y coordinate though, should be 16, will fix that. Will also change the rotations to number values, they're strings as a remnant of the CSS syntax having the `deg` suffix.